src/utils/entitySpec.js

```diff
@@ -103,6 +103,10 @@ export default class EntitySpec {
     return new EntitySpec(o.type, o.provider, o.namespace, o.name)
   }
 
+  static asPrLess(o) {
```

Comment:
`fromObject` would be more generically useful. I'll change that in another PR as this is merged